daemon/exec.go

```diff
@@ -87,6 +88,14 @@ func (d *Daemon) ContainerExecCreate(config *runconfig.ExecConfig) (string, erro
 	cmd := stringutils.NewStrSlice(config.Cmd...)
 	entrypoint, args := d.getEntrypointAndArgs(stringutils.NewStrSlice(), cmd)
 
+	keys := []byte{}
+	if config.DetachKeys != "" {
+		keys, err = term.ToBytes(config.DetachKeys)
+		if err != nil {
+			logrus.Warnf("Wrong escape keys provided (%s) using default : ctrl-p ctrl-q", config.DetachKeys)
```

Comment:
I'll add the err string as it says that the espace sequence provided is not supported.